src/components/StyleGuide/Typography.mdx

```diff
@@ -432,11 +1140,58 @@ prepositions do not get capital letters unless they start the title. For example
 </Box>
 }
 
+## Line Length and Line Height
+
+Wide lines of text are difficult to read and are proven to make it harder for
+people to focus, while short lines of text force the eye to travel back too
+often, breaking a reader's rhythm.
+
+For body text in desktop and tablet viewports, the recommended line length is
+between 60 and 75 characters. In a mobile viewport, the recommendation line
+length is between 30 to 40 characters.
+
+The testing to determine the ideal line length is generally performed using
+character count as the metric, but this is not the best approach when we are
+actually building websites. Rather, it is easier to look at the width of a text
+region as we are aiming for ideal sizing and spacing.
+
+NOTE: Yes, the content area in Storybook is wider than the NYPL recommendations.
+
+**Recommendations for Desktop**
+
+For desktop and tablet, when the font size is set to 16px (the default here at
+NYPL), the recommended width of text regions is 640px to 720px.
+
+**Recommendations for Mobile**
+
+For mobile, using the default 16px font size, the recommended width of text
+regions is 300px to 380px.
+
+**Line Height**
+
+The line height values used in the NYPL typography styles are controlled by the
+Reservoir Design System, so NYPL designers will not need to consider this during
+the design process. However, all NYPL designers should understand the reasoning
+behind the choices that were made for the NYPL typography styles.
+
+Line height (the vertical space between lines of text, also known as “leading”
+or "line spacing") is also important for legibility. Much like line length and
+font size, line height also affects how people read. If it's too tight, the
+lines of text become hard to scan. Too loose and the lines of text appear to be
+unrelated to each other, which is confusing for readers.
+
+The perfect line height depends on the design and size of the font itself, so
+there is no magic number that works for all text. That being said, a line height
+of 1.5 times the font size is always a good rule of thumb for body text.
+
 ## Character Count
 
+Content creators should be mindful of the character count in the following text
```

Comment:
Just curious, when you say content creators regarding our site, do you imagine this will be for static pages? I imagine this _might_ be different for blog creators or press releases? NYPL has a blog editorial guide and I don't see similar char count.

Reply:
These are the new guidelines, so there may be plenty of placces where the text does not yet align with these recommendations.  

Do you have a link to the blog editoria guide?  I've never seen it.  And I doubt Lily (who compiled these recommendations) has seen it either.